ITKImageProcessingFilters/itkImageToDream3DDataFilter.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _ITKImageToDream3DDataFilter_h
+#define _ITKImageToDream3DDataFilter_h
+
+#include "itkImage.h"
+#include "itkProcessObject.h"
+
+#include "SIMPLib/DataContainers/DataContainerArray.h"
+
+
+namespace itk
+{
+	template<typename PixelType, unsigned int VDimension>
+	class ImageToDream3DDataFilter: public ProcessObject
+	{
+	public:
+		/** Standard class typedefs. */
+		typedef ImageToDream3DDataFilter                                Self;
+		typedef SmartPointer<Self>                                         Pointer;
+		typedef typename Image<PixelType, VDimension> ImageType;
+		typedef typename ImageType::Pointer ImagePointer;
+		typedef typename DataArray<PixelType> DataArrayPixelType;
+		/** Method for creation through the object factory. */
+		itkNewMacro(Self);
+		void SetDataArrayPath(DataArrayPath dataArray);
+		itkGetMacro(DataArrayPath, DataArrayPath);
+		virtual void SetInput(const ImageType *image);
+		void Update();
+		DataContainer::Pointer GetOutput();
+	protected:
+		void Check();
```

Comment:
For this and the other class, a more descriptive name for `Check` could help.  Maybe `ValidInputCheck` or ...